witness/src/lib.rs

```diff
@@ -214,3 +254,18 @@ impl<F: Sync + Send + Copy + PrimeCharacteristicRing> Index<usize> for RowMajorM
         &self.inner.values[num_col * idx..][..num_col]
     }
 }
+
+pub fn expand_from_coeff<F: Ord + TwoAdicField>(
```

Comment:
This util function
1. could be move back to mpcs crate as the dft operation is not very general to witness itself
2. need to clear document to use it carefully because it involve Radix2DitParallel twiddle initialization which is veryf slow